README.md

```diff
@@ -5,7 +5,6 @@
 
 ## BuildKit
 
-<!-- godoc is mainly for LLB stuff -->
```

Comment:
removed this because it shows up in https://hub.docker.com/r/tonistiigi/buildkit/